src/AzureAuth/Commands/Ado/CommandPat.cs

```diff
@@ -26,6 +38,29 @@ public class CommandPat
         private const string OutputOption = "--output";
         private const string OutputHelp = "How PAT information is displayed. [default: token]\n[possible values: none, status, token, base64, header, headervalue, json]";
 
+        private const string DomainOption = "--domain";
+        private const string DomainHelp = "The preferred domain used when acquiring Azure Active Directory access tokens. [default: microsoft.com]";

```

Comment:
Interesting! I did not know we could have a PAT generated for a non Microsoft domain.

Reply:
Presumably it could happen if you were using `azureauth ado pat` as part of a non-Microsoft organization.

Reply:
Umm, ADO is not Microsoft only and we sell to customers? Good to know! :)